py/h2o_wave/pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
     "Click",
     "inquirer",
     "httpx>=0.16.1",
+    "psutil>=5.9.7",
```

Comment:
Can this be done without external dep? Seems like this PR is using it for a single function anyway.